recipes/rg

```diff
@@ -0,0 +1,4 @@
+(rg
+ :fetcher github
+ :repo "dajva/rg.el"
+ :branch "melpa")
```

Comment:
Oh, ok. Seems a bit backwards to me but sure, I'll change it.
The question is to what? Is `master` always assumed to be a development branch or what do you expect there?